source/PreferencesPanel.cpp

```diff
@@ -64,9 +64,10 @@ namespace {
 	const string BOARDING_PRIORITY = "Boarding target priority";
 	const string BACKGROUND_PARALLAX = "Parallax background";
 	const string ALERT_INDICATOR = "Alert indicator";
+	const string DISCOVERY_MODE = DISCOVERY_MODE_LABEL;
```

Comment:
Dont use a label, just write it directly like for all the others
```suggestion
	const string DISCOVERY_MODE = "Outfitter Discovery"
```

Reply:
also discovery shouldnt have a maj D